Microsoft.DotNet.UpgradeAssistant.sln

```diff
@@ -1010,7 +1010,7 @@ Global
 		{DA7F5186-9971-4BC0-9BDB-AD5F9058C4C3} = {8B8F1470-B862-4BF8-B41D-F078D43277FA}
 		{8B8F1470-B862-4BF8-B41D-F078D43277FA} = {60D2D4E3-E055-4A9C-BDB1-7328DAA9BD5C}
 		{16141B3F-F785-4CFD-9FD2-67E9D76C4827} = {4E12F429-CEC0-4080-B0FE-39F9B8B9AE4C}
-		{61A3EF4A-68CC-442E-87EC-F05B874A4C98} = {EE464F98-73F3-4858-8B5C-5254EECFA3B3}
```

Comment:
The backup step tests were showing up under the samples node in the solution explorer for some reason, instead of under tests. This fixes that.